registry.k8s.io/images/k8s-staging-etcd/images.yaml

```diff
@@ -18,6 +18,7 @@
     "sha256:51eae8381dcb1078289fa7b4f3df2630cdc18d09fb56f8e56b41c40e191d6c83": ["3.5.7-0"]
     "sha256:b3a0da984f9af598234c018faa4f28dd6a6461f5719163a61f10ebf26dbca1e9": ["3.5.8-0"]
     "sha256:e013d0d5e4e25d00c61a7ff839927a1f36479678f11e49502b53a5e0b14f10c3": ["3.5.9-0"]
+    "sha256:": ["3.5.10-0"]
```

Comment:
Looks like the sha256 has not been added, refer lines above for 3.5.9 and earlier.